src/main/scala/org/broadinstitute/dsde/workbench/sam/Boot.scala

```diff
@@ -55,7 +54,6 @@ object Boot extends IOApp with LazyLogging {
 
     // we need an ActorSystem to host our application in
     implicit val system = ActorSystem("sam")
-    implicit val materializer = ActorMaterializer()
```

Comment:
akka 2.6 makes the ActorMaterializer implicit for free with an ActorSystem
https://doc.akka.io/docs/akka/current/project/migration-guide-2.5.x-2.6.x.html#materializer-changes

> Materializer changes
> A default materializer is now provided out of the box. ... for Scala an implicit materializer is provided if there is an implicit ActorSystem available. This avoids leaking materializers and simplifies most stream use cases somewhat.
> 
> The ActorMaterializer factories has been deprecated and replaced with a few corresponding factories in akka.stream.Materializer. New factories with per-materializer settings has not been provided but should instead be done globally through config or per stream, see below for more details.
> 
> Having a default materializer available means that most, if not all, usages of Java ActorMaterializer.create() and Scala implicit val materializer = ActorMaterializer() should be removed.